content/05.conclusions.md

```diff
@@ -3,4 +3,53 @@
 We implemented AI-based models into publishing infrastructure.
 While most manuscripts have been written by humans, the process is time consuming and academic writing can be difficult to parse.
 We sought to develop a technology that academics could use to make their writing more understandable without changing the fundamental meaning.
-This work lays the foundation for a future where academic manuscripts are constructed by a process that incorporates both human and machine authors.
\ No newline at end of file
+This work lays the foundation for a future where academic manuscripts are constructed by a process that incorporates both human and machine authors.
+
+
+We found that, in general, the tool generated reasonable suggestions for most sections of the manuscripts, with paragraph that were greatly improved and others that needed minimal human review.
+We also found, though, that for some paragraph the revision was not good enough, and in some cases it even made the text worse.
+This was particularly true for short paragraphs and some in the the Methods sections when mathematical expressions were present.
+
+Abstracts:
```

Comment:
I think that going into detail for the sections is _way_ too much for a conclusions section. I'd recommend skipping these portions.